rancher/ui
# Update linkify

Contributes to https://github.com/rancher/dashboard/issues/8067

Updates `linkify` to newest version (4.1.3).

Note the import for `linkify` changed - instead of `linkify/string`, there is a package `linkity-string`

To test:

- In a project in the local cluster
- Create a secret and add a description of `test https://www.rancher.com`
- From the secrets list, click on the secret just created
- Verify that the description renders as a banner and that the text has been linkified, i.e:
- Ensure that the link opens in a new tab
![image](https://github.com/rancher/ui/assets/1955897/eb47767f-73de-40ac-a239-1ae45837054a)
